### PR TITLE
Quiz app to radix namespace

### PR DESCRIPTION
**Pull request by ElijahAhianyo**

### CODE REVIEW

File: quiz/quiz/quiz.py [hunk 1]
```diff
@@ -0,0 +1,170 @@
+"""Welcome to Reflex! This file outlines the steps to create a basic app."""
+import reflex as rx
+import copy
+from .results import results
+from typing import Any
+from typing import List
```
Comment (martinxu9) [inline quiz/quiz/quiz.py:6]: nit: can import both from the same import statements

File: quiz/quiz/quiz.py [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+
+    @rx.var
+    def percent_score(self):
+        return f"{self.score}%"
+
+
+def header():
+    return rx.vstack(
+        rx.heading("Python Quiz"),
+        rx.radix.separator(),
```
Comment (martinxu9) [inline quiz/quiz/quiz.py:51]: `rx.divider`?

File: quiz/quiz/quiz.py [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+    )
+
+
+def question1():
+    """The main view."""
+    return rx.vstack(
+        rx.heading("Question #1"),
+        rx.text(
+            "In Python 3, the maximum value for an integer is 26",
+            rx.chakra.text("3", as_="sup"),
```
Comment (martinxu9) [inline quiz/quiz/quiz.py:64]: still has chakra text here

File: quiz/rxconfig.py [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import reflex as rx
+
+config = rx.Config(
+    app_name="quiz",
+    db_url="sqlite:///reflex.db",
```
Comment (martinxu9) [inline quiz/rxconfig.py:5]: these two lines are not needed?

**Review by martinxu9 — CHANGES_REQUESTED** [review]
can we also switch to the new favicon.ico, the one right now is the pynecone one

File: quiz/quiz/quiz.py (hunk 3 above)
Comment (ElijahAhianyo) [reply quiz/quiz/quiz.py]: Raised this PR(https://github.com/reflex-dev/reflex/pull/2620/files) to fix the issue with radix text component as `as_="sup"` wasnt supported.

**Review by masenf — APPROVED** [review]

**Pull request merged**